app/views/users/register.html.haml

```diff
@@ -0,0 +1,12 @@
+%section.register.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      = bootstrap_form_for @user, layout: :horizontal, label_col: 'col-sm-2', control_col: 'col-sm-6', html: { class: 'new_user form-horizontal'} do |f|
```

Comment:
You shouldn't need the `form-horizontal` class explicitly written here. I'm not sure you need `new_user` either actually.